doc/source/python/python_component.md

```diff
@@ -221,6 +221,72 @@ class UserCustomException(Exception):
 
 ```
 
+## Gunicorn and load
+
+The wrapped python class will be run under [gunicorn](https://gunicorn.org/). As part of initialization of each gunicorn worker a `load` method will be called on your class if it has it. You should use this method to load and initialise your model. This is important for Tensorflow models which need their session created in each worker process. The [Tensorflow MNIST example](../examples/deep_mnist.html) does this.
+
+```
+import tensorflow as tf
+import numpy as np
+import os
+
+class DeepMnist(object):
+    def __init__(self):
+        self.loaded = False
+        self.class_names = ["class:{}".format(str(i)) for i in range(10)]
+        
+    def load(self):
+        print("Loading model",os.getpid())
+        self.sess = tf.Session()
+        saver = tf.train.import_meta_graph("model/deep_mnist_model.meta")
+        saver.restore(self.sess,tf.train.latest_checkpoint("./model/"))
+        graph = tf.get_default_graph()
+        self.x = graph.get_tensor_by_name("x:0")
+        self.y = graph.get_tensor_by_name("y:0")
+        self.loaded = True
+        print("Loaded model")
+        
+    def predict(self,X,feature_names):
+        if not self.loaded:
+            self.load()
+        predictions = self.sess.run(self.y,feed_dict={self.x:X})
+        return predictions.astype(np.float64)
+```
+
+### Gunicorn workers
+
+By default 4 gunicorn workers will be created for your model. If you wish to change this default then add the environment variable GUNICORN_WORKERS to the container for your model. An example is shown below:
```

Comment:
If you are using gunicorn workers and make this variable overridable,  you probably also need to consider (1) the `preload` option to reduce the memory usage, and (2) `max_requests` (probably along with `max_requests_jitter`) to restart the workers to avoid their memory leak. These are the opts in gunicorn command, but I guess there should be corresponding APIs that provide those options too.

Ref: http://docs.gunicorn.org/en/stable/settings.html

Reply:
I think we would need to make peload optional as not sure it will work for Tensorflow graphs that need a separate session in each fork.